src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +292,31 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                try:
+                    should_get_codecov_line_coverage = (
+                        features.has(
+                            "organizations:codecov-stacktrace-integration",
+                            project.organization,
+                            actor=request.user,
+                        )
+                        and project.organization.flags.codecov_access
+                    )
+                    if should_get_codecov_line_coverage:
+                        (
+                            result["lineCoverage"],
+                            result["codecovStatusCode"],
+                        ) = self.get_codecov_line_coverage(
+                            current_config["config"]["repoName"],
+                            current_config["config"]["provider"]["key"],
+                            current_config["config"]["defaultBranch"],
+                            result["sourceUrl"],
+                            CODECOV_TOKEN,
+                        )
+                except Exception:
+                    if not result.get("codecovStatusCode") or result["codecovStatusCode"] != 404:
```

Comment:
Why this check instead of just the `logger.exception`?

Reply:
I think a 404 from Codecov corresponds to an org with no Codecov integration, so I had suggested we handle that separately.

Reply:
Ah right. Good idea.